setup.py

```diff
@@ -27,4 +27,5 @@
                'sdist': {'dist_dir': '_dist'}},
 
       install_requires=['isodate >= 0.4.9',
-                        'requests >= 2.0.0'],)
+                        'requests >= 2.0.0',
```

Comment:
On Mon, Feb 01, 2016 at 04:58:01AM -0800, Ina Panova wrote:
> I read on pypi that minimum tested version of requests was 2.1.0 so would not it be better to
> raise the version for safety?

The required versions should be used to specify the API versions we need
from our dependencies, so I think it is good to leave it at 2.0.0 if
that is the API we need. Keep in mind that this could be important for
third party packagers who may be packaging Pulp for other distributions
in the future and may not have access to 2.1.0 in another distribution.

-- 
Randy Barlow
irc:   bowlofeggs